std_msgs/Header header
int16[5] ch_x
uint8 s1
uint8 s2
int16 mouse_x
int16 mouse_y
int16 mouse_z
uint8 mouse_press_left
uint8 mouse_press_right
uint16 key_value
int16[2] virtual_rocker